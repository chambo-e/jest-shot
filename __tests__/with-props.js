import shot from '../src/';
import WithProps from '../__components__/with-props';

const mutations = [
	{
		description: 'without props, should snap default props',
	},
	{
		description: 'with custom props',
		props: {
			snapPhrase: 'test snapshot 1',
		},
	},
	{
		// Without description
		props: {
			snapPhrase: 'should be snapshoted',
		},
	},
];

shot(WithProps, mutations);
