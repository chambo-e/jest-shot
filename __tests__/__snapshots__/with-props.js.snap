// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentWithProps mutation nÂ°2 1`] = `
<div>
  should be snapshoted
</div>
`;

exports[`ComponentWithProps with custom props 1`] = `
<div>
  test snapshot 1
</div>
`;

exports[`ComponentWithProps without props, should snap default props 1`] = `
<div>
  Should only be snapshoted when no props
</div>
`;
